src/main/java/com/programmers/domain/Book.java

```diff
@@ -0,0 +1,102 @@
+package com.programmers.domain;
+
+import java.util.List;
+
+public class Book {
+    @Override
+    public String toString() {
+        return "Book{" +
+                "bookId=" + bookId +
+                ", title='" + title + '\'' +
+                ", author='" + author + '\'' +
+                ", totalPageNumber=" + totalPageNumber +
+                ", bookStatus=" + bookStatus +
+                '}';
+    }
+
+    private Long bookId;
+    private String title;
+    private String author;
+    private int totalPageNumber;
```

Comment:
```suggestion
    private final String title;
    private final String author;
    private final int totalPageNumber;
```

크게 변경사항이 없는 필드로 보여집니다. 이러한 필드는 final 로 선언하여 최대한 불변 객체가 만들어질 수 있도록 하는 것이 좋습니다.

Reply:
[[refactor] 9. 변경사항이 없는 필드 final 제어자 추가](https://github.com/prgrms-be-devcourse/java-library-management/commit/637b1755aa06b015ba4b8734184c189163a8858a)

반영 완료했습니다!